application/forms/AddNodeForm.php

```diff
@@ -88,6 +88,18 @@ protected function addNewProcess()
                 'This is the unique identifier of this process'
             ),
             'validators'    => [
+                [
+                    'validator' => 'Regex',
+                    'options'   => [
+                        'pattern'   => '/^[a-zA-Z0-9](?:[\w\h.-]*)?[a-zA-Z0-9]$/',
```

Comment:
This still allows a trailing underscore. Just did a test and hosts/services as well as processes can have a trailing underscore while on master. What's the issue with it?
https://github.com/Icinga/icingaweb2-module-businessprocess/blob/da2ed1aef4848bfa8194f40de423337f9d4373e6/application/forms/BpConfigForm.php#L29